contact/templates/contact/submit-success.html

```diff
@@ -0,0 +1,35 @@
+{% extends "core/base.html" %}
+{% load static from staticfiles %}
+{% load message_box from directory_components_tags %}
+
+{% block head_css %}
+    {{ block.super }}
+    <link href="{% static 'styles/contact.css' %}" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block content %}
+
+    {% include 'contact/breadcrumbs.html' %}
+    <div class="container">
```

Comment:
can we remove this outer container div since there's already one in the section tag?